de.metas.serviceprovider/src/main/java/de/metas/serviceprovider/milestone/Milestone.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * #%L
+ * de.metas.serviceprovider.base
+ * %%
+ * Copyright (C) 2019 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.serviceprovider.milestone;
+
+import de.metas.organization.OrgId;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NonNull;
+
+import javax.annotation.Nullable;
+import java.time.LocalDateTime;
+
+@Data
+@Builder
+public class Milestone
+{
+	@NonNull
+	private OrgId orgId;
+
+	@Nullable
+	private MilestoneId milestoneId;
+
+	@Nullable
+	private String externalId;
+
+	@Nullable
+	private String description;
+
+	@NonNull
+	private String name;
+
+	@NonNull
+	private String value;
+
+	@Nullable
+	private LocalDateTime dueDate;
```

Comment:
why not ZonedDateTime?

Reply:
no exact reason, `LocalDateTime` was enough for what I needed.

Reply:
IMHO `dueDate` is a logistic date,  so it has to be precise, so timezone is needed for that.

LocalDateTime is a date+time but without a time zone.

If for example, the due date is "2020-04-20 13:14" it's important to know in which timezone, CEST, EEST?

If time is not important, I would suggest using LocalDate.

Reply:
I see, since time is not set for dueDate, I'll go with LocalDate, thx.